modules/ROOT/pages/reference/procedures.adoc

```diff
@@ -1006,7 +1006,9 @@ m|READ
 |===
 | Description
 a|
-List all available labels in the database.
+List available node labels within the database.
```

Comment:
Do these not get autogenerated from the source anymore? Should it be updated there too?

Reply:
We no longer autogenerate anything from the source. And yes, we need to keep the source and the docs in sync, so that we are able to add a test to catch updates at some point.